src/pages/state-management.mdx

```diff
@@ -128,23 +128,27 @@ However, developers often do reach for global state manager libraries too soon.
 Now that you understand the differences between local and global state - you need to understand how far you can take local state before introducing a global store in your application.
 
 - [Choosing the State Structure](https://react.dev/learn/choosing-the-state-structure) - Advice for how to structure the specific data you will be managing
+- [Extracting State into a Reducer](https://react.dev/learn/extracting-state-logic-into-a-reducer)
 - [Lifting State](https://react.dev/learn/sharing-state-between-components#lifting-state-up-by-example) - Lifting state to a parent can sometimes alleviate the need for global stores
     - [Update the State of a Sibling Component by Lifting State](https://alexsidorenko.com/blog/react-siblings/)
     - [Avoiding Prop-Drilling by Lifting State](https://alexsidorenko.com/blog/react-prop-drilling-composition/)
-- [Extracting State into a Reducer](https://react.dev/learn/extracting-state-logic-into-a-reducer)
-- [State Co-Location](https://kentcdodds.com/blog/application-state-management-with-react) - what is co-location & how it can make your app [faster](https://kentcdodds.com/blog/state-colocation-will-make-your-react-app-faster)
 
 ## When Do You Need Global State?
 
 1. If you&apos;re still prop drilling a lot (after trying the above approaches to lifting state) or feel you&apos;re headed in that direction and want to avoid it
 2. If you have some type of state/store that lives outside of your React application, but you need to consume it in your React app
 
-If you&apos;ve come this far and now you know you need a global state - the next question is - what are your options? Here are three popular approaches you can take (in order of complexity):
+If you&apos;ve come this far and now you know you need a global state - the next question is - what are your options? Here are the two popular approaches you can take:
 
-1. "Smart" Data Fetching (tanstack-query, rtk-query, swr)
-2. Contexts (`useContext` + `useReducer`)
+1. Data Fetching & Server Cache (tanstack-query, rtk-query, swr)
 3. Global Stores (Redux, Recoil, etc.)
 
+### Why Not Context?
+
+While many React educators on the web and even the React docs recommend [Context](https://react.dev/learn/passing-data-deeply-with-context) for global-like state management - we find that unless your application is very simple and will remain very simple - context is a very limiting state management solution.
+
+Read more in our upcoming advanced State Management Guide why we think there are better options, but also when Context IS the best tool for the job.
+
```

Comment:
[This article](https://blog.testdouble.com/posts/2021-03-19-react-context-for-dependency-injection-not-state/) would be amazing to link here. Suggested text:

> The [React docs](https://react.dev/learn/passing-data-deeply-with-context) explain how to use React Context for built-in global-like state management. While this can be a suitable solution for simple applications, we find that using Context correctly is more difficult and limiting than using the tools suggested in this guide.
> 
> Our upcoming advanced State Management Guide will explain how to use Context correctly, when it's a good choice, and why we think there are [better options](https://blog.testdouble.com/posts/2021-03-19-react-context-for-dependency-injection-not-state/).